addons/binding/org.openhab.binding.lametrictime/ESH-INF/thing/generic-app.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I was thinking more about the generic apps - there is some value in being able to at least select them as the active app on the device. What do you think about leaving them in, but removing the `frames` channel for now?

Reply:
Never mind what I said here. When implementing the state provider, I was considering using the things rather than the apps actually on the device, but I scrapped that idea. We can remove this without affecting the selection list.